lib/components/DeviceSelection.jsx

```diff
@@ -36,15 +36,35 @@ var DeviceSelection = React.createClass({
     var self = this;
 
     var items = _.map(this.props.uploads, function(upload) {
+
+      var isChecked = _.contains(self.props.targetDevices, upload.key);
+      var driverLink = '';
+
+      if((window.app.state._os === 'mac') && (upload.mac !== undefined) ) {
+        driverLink = upload.mac.driverLink;
+      }
+
+      if((window.app.state._os === 'win') && (upload.win !== undefined) ) {
+        driverLink = upload.win.driverLink;
+      }
+
+      var displayText = '';
+      if (isChecked && (driverLink !== '')) {
```

Comment:
I'm going to channel @jh-bate here and ask you to use `!_.isEmpty(driverLink)` - that's kind of our standard.